src/Microsoft.AspNet.Razor/Parser/TagHelpers/TagHelperParseTreeRewriter.cs

```diff
@@ -70,7 +70,7 @@ private void RewriteTags(Block input, RewritingContext context)
                 // At this point the child is a Span or Block with Type BlockType.Tag that doesn't happen to be a
                 // tag helper.
 
-                // Add the child to current block. 
+                // Add the child to current block.
```

Comment:
I know I can ignore these changes. but you're making enough of them that they should be in a separate "cleanup trailing whitespace" PR that you commit immediately.